### DISCUSSION TITLE
How to create a redirect

### DISCUSSION THREAD

**Discussion by thecraftman** [mkdocs/mkdocs#2453] [Q&A]
on our docs page, I want to create a redirect that links to https://docs.axiom.co/cli the actual link is https://docs.axiom.co/reference/CLI/

When I configure strict: true it still doesn't work. i used the  redirect plugins - redirects: redirect_maps: and it didn't point to the URL. 

@waylan @d0ugal @BetterThanReal @tomchristie @Hernrup @mandel01

**Comment by waylan** [comment]
I would suggest asking on the plugin project. They will have more intimate knowledge of how the plugin works.

**Reply by thecraftman** [reply]
Okay, Thanks @waylan. Please can you send a link to the plugin project.

**Reply by waylan** [reply]
I don't know which plugin you are using. How would I know what the link is?

**Reply by Andre601** [reply]
It's https://github.com/datarobot/mkdocs-redirects since they mentioned the `redirects_maps` which is a plugin option.
Source: I use it myself

**Reply by Andre601** [reply]
It's btw also listed in the plugins page of the Wiki

**Reply by lovelydinosaur** [reply]
Right, so you probably need to ask them about it instead... https://github.com/datarobot/mkdocs-redirects/issues

**Reply by thecraftman** [reply]
okay, will do. @Andre601 can i reach out to know how you configured it. when you have the chance?

**Reply by Andre601** [reply]
https://github.com/Andre601/blog/blob/master/mkdocs.yml#L214-L219

**Reply by thecraftman** [reply]
Thanks @Andre601
